src/views/Chats.vue

```diff
@@ -29,10 +29,10 @@
                 <div class="md-list-text-container">
                     <div class="chat_entry_title">{{ chatName(chat.partner) }}</div>
                     <div class="chat-entry-wrapper">
-                      <div v-if="(chat.last_message.amount > 0 && chat.last_message.direction === 'from') || chat.last_message.direction === 'from'" class="message-status-block" :message-status="chat.last_message.confirm_class"></div>
-                      <chat-entry v-if="chat.last_message.amount > 0 && chat.last_message.direction === 'from'" :message="chat.last_message" :brief="true"></chat-entry>
+                      <div v-if="messageStatusCondition(chat.last_message)" class="message-status-block" :message-status="chat.last_message.confirm_class"></div>
```

Comment:
I believe we can calculate and display status inside the `<chat-entry />` since it has all the required information and is actually supposed to encapsulate chat entry display.